internal/liquidity/lp_test.go

```
@@ -104,21 +104,25 @@
 	// Create a mock HTTP server
 	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		// Create a mock response
-		response := map[string]interface{}{"stakedAmount": "170", "reward": "1.5"}
+		response := map[string]interface{}{
+			"jsonrpc": "2.0",
+			"id":      6826472344863318,
+			"result":  "0x00000000000000000000000000000000000000000000000ad78ebc5ac62000000000000000000000000000000000000000000000000000000000000000000000",
+		}
 		w.WriteHeader(http.StatusOK)
 		json.NewEncoder(w).Encode(response)
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)